src/main/java/com/opinta/entity/Parcel.java

```diff
@@ -0,0 +1,52 @@
+package com.opinta.entity;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Cascade;
+import org.hibernate.annotations.CascadeType;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+import org.hibernate.annotations.LazyCollection;
+import org.hibernate.annotations.LazyCollectionOption;
+
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import java.math.BigDecimal;
+import java.util.List;
+
+@Entity
+@Data
+@NoArgsConstructor
+public class Parcel {
+    @Id
+    @GeneratedValue
+    private long id;
+    private float weight;
+    private float length;
+    private float width;
+    private float height;
+    private BigDecimal declaredPrice;
+    private BigDecimal price;
+    @OneToMany(fetch = FetchType.EAGER)
+    @LazyCollection(LazyCollectionOption.FALSE)
+    @Fetch(value = FetchMode.SUBSELECT)
+    @Cascade(CascadeType.SAVE_UPDATE)
```

Comment:
This is how i fixed the problem that Hibernate didn't saved ParcelItems to db. Adding save, solved a problem.